cg/server/endpoints/applications.py

```diff
@@ -23,6 +22,14 @@ def get_applications():
     return jsonify(applications=parsed_applications)
 
 
+@APPLICATIONS_BLUEPRINT.route("/applications/<order_type>")
+@handle_endpoint_errors
+def get_application_order_types(order_type: str):
```

Comment:
Can this also be type hinted as OrderType already?

Reply:
wish granted